src/common/hooks/useHandleOrderUpdateError/useHandleOrderUpdateError.js

```diff
@@ -32,6 +37,11 @@ const useHandleOrderUpdateError = (mutatorExpenseClass) => {
             values,
           });
         }
+      } else if (errorMessage?.includes(INVALID_TOKEN_MESSAGE)) {
```

Comment:
Both this condition and the one above handle errors, so it's weird to have one inspect `errorCode` and the other `errorMessage`. Could they be structure the same way?

Reply:
No, it is not possible. According to the BE team, the error comes from the authorization module and we need to parse the error message to detect the error message.

Reply:
I understand that top-level error codes like 401 or 403 might be caused by multiple different reasons, but it feels surprising/unfortunate/wrong that our _own_ error codes that we control cannot be more specific. I understand that you are just dealing with the API you've been given, but I would push back against the BE team and ask for a more expressive API.

Reply:
@zburke updated